pkg/controllers/dynakube/connectioninfo/oneagent/reconciler.go

```diff
@@ -43,7 +43,7 @@ func NewReconciler(clt client.Client, apiReader client.Reader, dtc dtclient.Clie
 var NoOneAgentCommunicationHostsError = errors.New("no communication hosts for OneAgent are available")
 
 func (r *reconciler) Reconcile(ctx context.Context) error {
-	if !r.dk.NeedAppInjection() && !r.dk.NeedsOneAgent() {
+	if !r.dk.NeedAppInjection() && !r.dk.NeedsOneAgent() && !r.dk.Spec.LogModule.Enabled {
```

Comment:
can you create a method, similar to the others we have, like `NeedsLogModule()` to abstract the Dynakube access?